tests/test_automation/github-actions/ci/run_step_ornl-nitrogen-1.sh

```diff
@@ -0,0 +1,79 @@
+#!/bin/bash
+
+# Executes pipeline steps for the nitrogen-1 CI jobs
```

Comment:
Should we also move nitrogen-cuda setting also here?

Reply:
I will do a separate PR for ornl-nitrogen-2.sh